packages/neo-one-cli/src/interactive/createCRUD.js

```diff
@@ -151,22 +153,24 @@ const renderTasks = (
 };
 
 const promptDelete = ({
-  cli,
   crud,
   name,
 }: {|
-  cli: InteractiveCLI,
   crud: CRUDResource<*, *>,
   name: string,
-|}) =>
-  cli.vorpal.activeCommand.prompt({
```

Comment:
Actually, let's add `prompt` to the `InteractiveCLI` object directly, in case we need to change it again in the future. Add a method to `InteractiveCLI` that just delegates to `inquirer.prompt`.